kolibri/plugins/registry.py

```diff
@@ -30,8 +30,8 @@
 
 .. note::
 
-    Each app in ``INSTALLED_APPS`` is searched for the special
-    ``kolibri_plugin`` module.
+    Each app in ``ACTIVE_PLUGINS`` in the kolibri conf is searched for the
+    special ``kolibri_plugin`` module.
```

Comment:
`INSTALLED_APPS` is still the manifest of active Django applications - it just isn't the manifest of Kolibri plugins (although it will include all Kolibri plugins).

I have added warnings and other catches to warn and error when these assumptions are being violated. Can add some more test cases to extend these.